### DISCUSSION TITLE
Extension activation failed: "read ECONNRESET"

### DISCUSSION THREAD

**Discussion by henriquearthur** [community/community#13758] [Copilot Conversations] [answered]
I installed Copilot on VS Code and IntelliJ IDEA and it's not working on both IDEs. I'm behind a corporate proxy with authentication, it is properly configured on VS Code - other features that require internet connection work fine, but Copilot doesn't.

It seems to be proxy-related, but I don't know what to do.

The log on View > Output:
[ERROR] [default] [2022-03-29T11:32:32.228Z] GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"

The proxy on VS Code is configured on the settings `http.proxySupport`, `http.proxy` and `http.proxyStrictSSL`. It is also configured on Windows environment variables `http_proxy` and `https_proxy`.

If Copilot doesn't support proxy, can you share the URLs that it connects to so I can request a firewall rule for bypassing proxy?

VS Code v1.65.2
Copilot v1.9.5350

**Comment by davecheney** [comment, marked as answer]
Sadly copilot does not support proxies at the moment.

**Comment by davecheney** [comment]
Duplicate of https://github.com/github/feedback/discussions/13743
